test/hotspot/jtreg/vmTestbase/nsk/jdwp/ThreadReference/ForceEarlyReturn/forceEarlyReturn002/libforceEarlyReturn002a.cpp

```diff
@@ -23,23 +23,44 @@
 
 #include "jni.h"
 
+#if defined(_WIN32)
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
 extern "C" {
 
+static volatile bool wait_in_native = true;
```

Comment:
The wait_in_native should be atomic or used inside raw monitors since it is used by different thread.

Reply:
Fixed.